classes/image_type_tezi_ot3.bbclass

```diff
@@ -172,8 +172,8 @@ def rootfs_tezi_emmc(d, use_bootfiles):
 
     filesystem_partitions.append(
           {
-            "partition_size_nominal": 512,
-            "want_maximised": True,
+            "partition_size_nominal": 3000,
```

Comment:
Please use a power of 2, I think this is a binary value

Reply:
Changed the size to 2048, current rootfs is way under.